emmet-core/emmet/core/classical_md.py

```diff
@@ -0,0 +1,74 @@
+"""Schemas for classical MD package."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Optional
+
+from monty.json import MSONable
+
+from emmet.core.vasp.task_valid import TaskState
+from pydantic import BaseModel, Field
+
+
+@dataclass
+class MoleculeSpec(MSONable):
+    """A molecule schema to be output by OpenMMGenerators."""
+
+    name: str
+    count: int
+    formal_charge: int
+    charge_method: str
+    openff_mol: str  # a tk.Molecule object serialized with to_json
+
+    # @field_validator("openff_mol")
+    # @classmethod
+    # def _validate_openff_mol(cls, v: str) -> str:
+    #     try:
+    #         tk.Molecule.from_json(v)
+    #     except Exception as e:
+    #         raise ValueError(
+    #             "MoleculeSpec.openff_mol must be able to be"
+    #             "parsed with Molecule.from_json."
+    #         ) from e
+    #     return v
+
+
+class ClassicalMDTaskDocument(BaseModel, extra="allow"):  # type: ignore[call-arg]
+    """Definition of the OpenMM task document."""
+
```

Comment:
Would it make any sense to have an `ensemble` field here? If I recall correctly OpenMM doesn't necessarly enforce a strict ensemble so maybe / maybe not, but thinking generally about classical MD, that's the only significant piece of info that I don't see reflected in the schema.